src/extension.ts

```diff
@@ -44,7 +44,7 @@ function export_breakpoints(context: vscode.ExtensionContext) {
 					// outputChannel.appendLine(JSON.stringify(bp));
 					const start_line = bp.location.range.start.line;
 					const path = bp.location.uri.path;
-					bp_lines = bp_lines + "break " + path + ":" + start_line + "\n"
+					bp_lines = bp_lines + "break " + path + ":" + start_line + "\n";
```

Comment:
`;` is needed? (I'm TS newbe)

Reply:
It's not strictly needed, but the rest of the code had it so my formatter added it in. I can change my formatter to remove all `;`

Reply:
I also don't care about it.

Reply:
If you'd like you can create a config file called `.prettierrc.js` with the following [options](https://prettier.io/docs/en/options.html):

```
module.exports = {
  semi: false,
}
```

Then you can install this plugin: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

It works a lot like rubocop, formating your code to idiomatic typescript.

Reply:
Note: You would also need to change this eslint rule: `"@typescript-eslint/semi": "warn",` -> `"@typescript-eslint/semi": "off",`